store/searchtest/channel_layer.go

```diff
@@ -70,100 +70,100 @@ func TestSearchChannelStore(t *testing.T, s store.Store, testEngine *SearchTestE
-	require.Nil(t, apperr)
+	require.NoError(t, apperr)
 	th.checkChannelIdsMatch(t, []string{th.ChannelBasic.Id, alternate.Id}, res)
 }
 
 func testAutocompleteChannelByNameSplittedWithUnderscoreChar(t *testing.T, th *SearchTestHelper) {
 	alternate, err := th.createChannel(th.Team.Id, "channel_alternate", "ChannelAlternate", "", model.CHANNEL_OPEN, false)
-	require.Nil(t, err)
+	require.NoError(t, err)
 	defer th.deleteChannel(alternate)
 	res, apperr := th.Store.Channel().AutocompleteInTeam(th.Team.Id, "channel_a", false)
-	require.Nil(t, apperr)
+	require.NoError(t, apperr)
 	th.checkChannelIdsMatch(t, []string{alternate.Id}, res)
 }
 
 func testAutocompleteChannelByDisplayNameSplittedByWhitespaces(t *testing.T, th *SearchTestHelper) {
 	alternate, err := th.createChannel(th.Team.Id, "channel-alternate", "Channel Alternate", "", model.CHANNEL_OPEN, false)
-	require.Nil(t, err)
+	require.NoError(t, err)
 
 	defer th.deleteChannel(alternate)
 	res, apperr := th.Store.Channel().AutocompleteInTeam(th.Team.Id, "Channel A", false)
-	require.Nil(t, apperr)
+	require.NoError(t, apperr)
 	th.checkChannelIdsMatch(t, []string{alternate.Id}, res)
 }
 func testAutocompleteAllChannelsIfTermIsEmpty(t *testing.T, th *SearchTestHelper) {
 	alternate, err := th.createChannel(th.Team.Id, "channel-alternate", "Channel Alternate", "", model.CHANNEL_OPEN, false)
-	require.Nil(t, err)
+	require.NoError(t, err)
 	other, err := th.createChannel(th.Team.Id, "other-channel", "Other Channel", "", model.CHANNEL_OPEN, false)
-	require.Nil(t, err)
+	require.NoError(t, err)
 	defer th.deleteChannel(alternate)
 	defer th.deleteChannel(other)
 	res, apperr := th.Store.Channel().AutocompleteInTeam(th.Team.Id, "", false)
-	require.Nil(t, apperr)
+	require.NoError(t, apperr)
 	th.checkChannelIdsMatch(t, []string{th.ChannelBasic.Id, alternate.Id, other.Id}, res)
 }
 
 func testSearchChannelsInCaseInsensitiveManner(t *testing.T, th *SearchTestHelper) {
 	alternate, err := th.createChannel(th.Team.Id, "channel-alternate", "ChannelAlternate", "", model.CHANNEL_OPEN, false)
-	require.Nil(t, err)
+	require.NoError(t, err)
 	defer th.deleteChannel(alternate)
 	res, apperr := th.Store.Channel().AutocompleteInTeam(th.Team.Id, "channela", false)
-	require.Nil(t, apperr)
+	require.NoError(t, apperr)
 	th.checkChannelIdsMatch(t, []string{th.ChannelBasic.Id, alternate.Id}, res)
 	res, apperr = th.Store.Channel().AutocompleteInTeam(th.Team.Id, "ChAnNeL-a", false)
-	require.Nil(t, apperr)
+	require.NoError(t, apperr)
 	th.checkChannelIdsMatch(t, []string{th.ChannelBasic.Id, alternate.Id}, res)
 }
 
 func testSearchOnlyPublicChannels(t *testing.T, th *SearchTestHelper) {
 	alternate, err := th.createChannel(th.Team.Id, "channel-alternate", "ChannelAlternate", "", model.CHANNEL_PRIVATE, false)
-	require.Nil(t, err)
+	require.NoError(t, err)
 	defer th.deleteChannel(alternate)
 	res, apperr := th.Store.Channel().AutocompleteInTeam(th.Team.Id, "channel-a", false)
```

Comment:
We should also change `apperr` to `err`.

Reply:
done